cmd/serve/s3/help.go

```diff
@@ -0,0 +1,45 @@
+package s3
+
+var longHelp = `
+Serve s3 implements a basic s3 server that serves a remote 
+via s3. This can be viewed with an s3 client, or you can make
+an s3 type remote to read and write to it.
+
+S3 server supports Signature Version 4 authentication. Just
+ use ` + `--s3-auth accessKey1-secretKey1,accessKey2-secretKey2` + ` and
+ set  Authorization Header correctly in the request. (See 
+https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html)
+
+Please note that some clients may require HTTPS endpoints. 
+See [#SSL](#ssl-tls) for SSL configuration.
+
+Use ` + `--host-bucket` + ` if you want to use bucket name as a part of 
```

Comment:
Is this what we call path style vs virtual host style in the s3 backend? If so we should probably name it similarly

      --s3-force-path-style   If true use path style access if false use virtual hosted style (default true)

Reply:
Changed. Currently using `--force-path-style`.